apps/sleeplog/settings.js

```diff
@@ -345,6 +347,19 @@
           writeSetting();
         }
       },
+      /*LANG*/"Wear Temperature": {
+        value: settings.wearTemp,
+        step: 0.5,
+        min: 20,
+        max: 40,
+        wrap: true,
+        noList: true,
+        format: v => v + "°C",
+        onchange: v => {
+          settings.wearTemp = v;
+          writeSetting();
+        }
+      },
```

Comment:
What do you think about merging this with the `Wear detection using temperature` setting? If the user sets it to the minimum value (which we could have as 19.5), it turns it off, then we just present a single menu item to them?

Reply:
Do you mean similar to the recent changes done to sched where if the lowest number gets selected it changes to "Forever" (in our case it would be "disabled") ?
```
// apps/sched/settings.js

    /*LANG*/"Buzz Count": {
      value: settings.buzzCount == null ? 4 : settings.buzzCount,
      min: 4,
      max: 15,
      step: 1,
      format: v => v === 4 ? "Forever" : v,
      onchange: v => {
        settings.buzzCount = v === 4 ? null : v;
        require("sched").setSettings(settings);
      }
    },
```
Sure, why not! :+1: I originally also wanted to implement it like this but thought this might make the user overlook the "disabled" option, but we can do it that way of course. I checked "Allow edits and access to secrets by maintainers" in this PR so if you want to, you can just edit it the way you want to.

Reply:
Yeah exactly that, well spotted! And thanks - I'll let you make the change for now but nice to know we can help out if needed :)